content/90.back-matter.md

```diff
@@ -328,46 +341,48 @@ Overall, we can recall a small amount of edges at high precision thresholds.
-| [G]{.gene_color}i[G]{.gene_color}       | SETBP1                  | SETBP1                    | 1.000                | 0.588                 | 354                 | the critical deleted region contains [setbp1]{.gene_color} gene ( [set binding protein 1 )]{.gene_color}.                                                                                                                                                                                                                                                       | 
-| [G]{.gene_color}i[G]{.gene_color}       | BCL2                    | BAK1                      | 0.118                | 0.587                 | 1220                | different expression patterns of [bcl-2]{.gene_color} family genes in breast cancer by estrogen receptor status with special reference to pro-apoptotic [bak]{.gene_color} gene.                                                                                                                                                                                | 
-| [G]{.gene_color}i[G]{.gene_color}       | SP1                     | INSR                      | 0.948                | 0.587                 | 23                  | thus , the efficient expression of the human [insulin receptor]{.gene_color} gene possibly requires the binding of transcriptional factor [sp1]{.gene_color} to four g-c boxes located -593 to -618 base pairs upstream of the atg translation initiation codon.                                                                                                | 
-| [G]{.gene_color}i[G]{.gene_color}       | ABCD1                   | ABCD1                     | 1.000                | 0.586                 | 410                 | x-linked adrenoleukodystrophy ( x-ald ) is caused by mutations in the [abcd1]{.gene_color} gene encoding the peroxisomal abc transporter adrenoleukodystrophy protein ( [aldp )]{.gene_color}.                                                                                                                                                                  | 
-| [G]{.gene_color}i[G]{.gene_color}       | CYP1A1                  | AHR                       | 0.996                | 0.586                 | 1940                | the liganded [ah receptor]{.gene_color} activates transcription by binding to a specific dna-recognition motif within a dioxin-responsive enhancer upstream of the [cyp1a1]{.gene_color} gene.                                                                                                                                                                  |
+#### Top Ten Sentences for Each Edge Type
```

Comment:
Quite a few of your top GiG edges are self-self edges. Would it make sense to exclude those?